analyzers/its/expected/Ember-MM/S1871-generic.EmberCore.Notifications.json

```diff
@@ -0,0 +1,28 @@
+{
+  "Issues": [
+    {
+      "Id": "S1871",
+      "Message": "Either merge this case with the identical one on line 113 or change one of the implementations.",
+      "Uri": "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/Ember-MM/Addons/generic.EmberCore.Notifications/Module.Notifications.vb#L115",
+      "Location": "Line 115 Position 25-38"
+    },
+    {
+      "Id": "S1871",
+      "Message": "Either merge this case with the identical one on line 113 or change one of the implementations.",
+      "Uri": "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/Ember-MM/Addons/generic.EmberCore.Notifications/Module.Notifications.vb#L117",
+      "Location": "Line 117 Position 25-38"
+    },
+    {
+      "Id": "S1871",
+      "Message": "Either merge this case with the identical one on line 113 or change one of the implementations.",
+      "Uri": "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/Ember-MM/Addons/generic.EmberCore.Notifications/Module.Notifications.vb#L119",
+      "Location": "Line 119 Position 25-38"
+    },
+    {
+      "Id": "S1871",
+      "Message": "Either merge this case with the identical one on line 113 or change one of the implementations.",
+      "Uri": "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/Ember-MM/Addons/generic.EmberCore.Notifications/Module.Notifications.vb#L121",
```

Comment:
Looks like a TN to me.

Reply:
Why?
All the `Case` clauses are the same and there is no `Else Case` clause.

It is a TP based on the exception of the exception of S1871:

```
But this exception does not apply to if chains without else-s, or to switch-es without default clauses when all branches have the same single line of code. In the case of if chains with else-s, or of switch-es with default clauses, rule [S3923](https://sonarsource.github.io/rspec#/rspec/S3923) raises a bug.
```